angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
+        '@angular-eslint/component-selector': 'error',
+        '@angular-eslint/contextual-decorator': 'error',
+        '@angular-eslint/directive-selector': 'error',
+        '@angular-eslint/no-attribute-decorator': 'error',
+        '@angular-eslint/no-forward-ref': 'error',
```

Comment:
[@angular-eslint/no-forward-ref](http://codelyzer.com/rules/no-forward-ref)

Reply:
ForwardRef is only used for specific, necessary problems.

**Proposal**
Leave this rule disabled.

Reply:
@mure this thread is awaiting your vote

Reply:
I'm willing and able to act as @mure 's proxy vote.

Reply:
I consent